man/systemd.timer.xml

```diff
@@ -49,9 +49,12 @@
 
     <para>Note that in case the unit to activate is already active at the time the timer elapses it is not restarted,
     but simply left running. There is no concept of spawning new service instances in this case. Due to this, services
-    with <varname>RemainAfterExit=</varname> set (which stay around continuously even after the service's main process
-    exited) are usually not suitable for activation via repetitive timers, as they will only be activated once, and
-    then stay around forever.</para>
+    with <varname>RemainAfterExit=true</varname> set (which stay around continuously even after the service's main
+    process exited) are usually not suitable for activation via repetitive timers, as they will only be activated
+    once, and then stay around forever. Targets, which by default do not deactivate on their own, can be activated
```

Comment:
I'd write "Target units" rather than just "Targets", but doesn't matter

Reply:
(at least the first time)

Reply:
Changed in both places, because it seemed more consistent to me.